dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1653,7 +1653,7 @@ private void printWesHelpFooter() {
         out("Global Optional Parameters:");
         out("  --wes-url <WES URL>                 URL where the WES request should be sent, e.g. 'http://localhost:8080/ga4gh/wes/v1'");
         out("");
-        out("NOTE: WES SUPPORT IS IN BETA AT THIS TIME. RESULTS MAY BE UNPREDICTABLE.");
+        out("NOTE: Currently only WDL workflows are supported for WES requests. Further language support is under development.");
```

Comment:
If AGC magically added support for CWL tomorrow, in theory the CLI would just work, wouldn't it?

That said, I think this is fine.

Reply:
It should, it just hasn't been tested.